src/dir.targets

```diff
@@ -157,9 +154,39 @@
     <DebugEngines>{2E36F1D4-B23C-435D-AB41-18E608940038}</DebugEngines>
   </PropertyGroup>
 
+  <Target Name="CheckTestCategories"
+          AfterTargets="Test"
+          Condition="'$(IsTestProject)'=='True' And '$(SkipTests)'!='True'">
+    <PropertyGroup>
+      <!-- By default project has inner- and outer- loop tests. -->
+      <TestCategories Condition="'$(TestCategories)'==''">InnerLoop;OuterLoop</TestCategories>
```

Comment:
We are going to very quickly get to a place where we need to have quick efficient ways of filtering out entire test projects and not always blindly running xUnit to let it determine there isn't anything to do. Right now we are only doing simple csc compiling but when we start introducing .NET Native test runs the cost of just compiling a test project to determine there isn't anything to do will become too high to pay. Perhaps a Roslyn based tool once we move to using Roslyn could be used to help default these values but I do believe we are going to need to have a way to specify them in the project file itself.